src/overview/design-goals.md

```diff
@@ -43,3 +44,26 @@ We want to follow some conservative design heuristics to minimize risk and mista
 - Rely as much as possible on design components that are already proven in production environments.
 - Rely as much as possible on design components with adequate theoretical underpinnings and security analyses.
 - Minimize changes or variations on the above: strive to only alter existing work when necessary for overall design goals. For example, Zcash's privacy or issuance constraints are likely less common among existing PoS designs.
+
+
+# Non-goals
+
+These are not goals of the TFL design, either to simplify the scope of the initial design (aka [Out of Scope Goals](#out-of-scope-goals)), or because we believe some potential goal _should not_ be supported (aka [Anti-goals](#anti-goals)).
+
+## Out of Scope Goals
+
+While these desiderata may be common across the blockchain consensus design space, they are not specific goals for the initial TFL design. Note that these may be goals for future protocol improvements.
+
+- Prioritizing minimal time-to-finality over other considerations (such as protocol simplicity, impact on existing use cases, or other goals above).
+- In-protocol liquid staking derivatives.
+- Maximizing the PoS staked-voter count ceiling. For example, [Tendermint BFT](https://github.com/tendermint/tendermint#research) has a relatively low ceiling of ~hundreds of staked voters, whereas Ethereum's [Gasper](https://ethereum.org/en/developers/docs/consensus-mechanisms/pos/gasper/) supports hundreds of thousands of staked voters.
```

Comment:
I believe having this as a non-goal may be incompatible with having finality via a BFT protocol + resistance to chain halts with attack cost higher than 24h of mining rewards, by the argument in https://github.com/Electric-Coin-Company/tfl-book/pull/105/files#r1379345913

Reply:
Hm, well, if we have an explicit goal (say the potentially implausibly high DDOS protection I proposed), then we shouldn't _preclude_ achieving any "out-of-scope goals" to achieve the in-scope goals.

We might think of goals as potentially having a dependency graph, and the explicit in-scope goals are a set of nodes, and they imply any dependencies to achieve them.

(By contrast we _would_ have to avoid an anti-goal, so anti-goals are constraining and should be defined carefully if at all.)

Reply:
For now I will merge, because I think we're going to want another overhaul of the entire goals section in the short to medium term anyway where we can address issues like this.